### PR TITLE
applications: nrf_desktop: Allow OS to suspend device drivers

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:DESK-664

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

**Comment by pdunaj** [comment]
We don't need to do any of this anymore as OS will cover the devices suspend from within `_sys_suspend`. As it is done atomically with going to deep sleep we will not have a problem with module accessing peripherals that were just suspended.

**Comment by pdunaj** [comment]
Tested on PCA20041 and works well

File: applications/nrf_desktop/src/modules/power_manager.c [hunk 1]
```diff
@@ -26,10 +26,10 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_POWER_MANAGER_LOG_LEVEL);
 
+
```
Comment (ppryga) [inline applications/nrf_desktop/src/modules/power_manager.c]: remove in future

**Comment by pdunaj** [comment]
Hi @wiminordic , @jhn-nordic , could you give a green tick?

**Pull request merged**